src/pal/src/locale/utf8.cpp

```diff
@@ -1399,7 +1399,6 @@ class UTF8Encoding
 
         EncodeChar:
 
-#ifdef FASTLOOP
```

Comment:
It looks like some removes of `#if (0) ` crept in in ilasm/**. I'll leave those removes.